tmt/steps/provision/testcloud.py

```diff
@@ -425,6 +427,7 @@ def start(self):
             raise ProvisionError(
                 f'Failed to boot testcloud instance ({error}).')
         self.guest = self.instance.get_ip()
+        self.port = self.instance.get_instance_port()
```

Comment:
That code means, that `port` is selected randomly by testcloud library. Or It is possible to force some port mapping e.g. `22 (SSH) -> 2201(localSSHport)` of first spawned machine? Why I'm asking is, that in case I would like to connect to test machine would have port numbers more predictable.  But not sure how implementation inside testcloud look like.

Reply:
There is a config option for testcloud: SSH_USER_PORT_BASE, which sets a start of used interval for ssh ports. It defaults to 10022 which means the first created vm will have that port, another will be 10023 and so on. It recycles ports that aren't in use, so eg. you create 3 vms, their ports will be 10022, 10023, 10024, you delete the second vm, create a new one and it'll have port 10023. I hope that answers the question.

Reply:
Thinking about this, there probably needs to be some change done to be able to change that from tmt, I can take a look at it.

Reply:
From the `tmt` point of view I would say we're set. The instance of the `Guest` class has the `port` available and can be used if necessary.

Reply:
Yep, it contains `self.port` as good solution, just when scheduled tests, sometimes would be nice to connect to the machine, and this info is not displayed in default mode, probably also not in verbose, just in debug mode probably, means, to kill the process and run it again.

Reply:
Good idea to show the `ip` and `port` to the user in verbose mode. Done in ac83c1a.

Reply:
@psss BTW would be good to display full ssh command line to connect to any machine instead of just printing port and host
so that I can imagine in vebose mode something like:
```
 f"Connect via: ssh -p {self.port} -i {self.ssh_key} {self.guest}" in case not used key, display also password
```
Theoretically I as an user I do not know where are the ssh keys or if there is some randomly choosed password for any instances: minute, virtual, connect, podman, ...

Reply:
We can add this to the debug log. But specifically for this use case `tmt run login` should provide an easy way to get to the box.